bitrise.yml

```diff
@@ -881,6 +881,21 @@ workflows:
         - scheme: Fennec
         - simulator_device: iPhone 8
     - deploy-to-bitrise-io@1.9: {}
+    - script@1:
+        title: Move glean sdk_generator back to MozillaAppServices
```

Comment:
I think this has to be moved to the RunSmokeXCUTest workflow... if you see the Bitrise build for this [PR](https://app.bitrise.io/build/da9b0e8949c117b3#?tab=log), it is still failing. I think that's why we run the RunUITest workflow and after it the SmokeXCUITest, if you see the steps in the Bitrise site, we are running this script before the second workflow run and so the script is not available for it...